src/bundling/Bundler.js

```diff
@@ -166,8 +166,17 @@ class Bundler {
      * a message row, date divider, or bundle row.
      */
     _calculateSortedTableRows(messageNodes, bundlesByLabel) {
+
+        chrome.storage.sync.get(['bundleDates'], ({ bundleDates = true }) => {
+            this.bundleDates = bundleDates;
+        });
+        
         const rows = this._calculateMessageAndBundleRows(messageNodes, bundlesByLabel);
 
+        if (!this.bundleDates) {
+            return rows;
+        }
+
```

Comment:
There's another place where the date grouping happens and this new logic is needed: https://github.com/teresa-ou/inboxy/blob/master/src/bundling/DateGrouper.js#L79

There's a star icon toggle that inboxy adds to the top right of the Gmail page -- turning that on, will trigger the DateGrouper code.

Reply:
Thanks for catching!